### PR TITLE
Update constraints.md

### PR DESCRIPTION
**Pull request by Denny-Hwang**
#163 

단어 논의 결과에 맞게 수정하였습니다.
세부 내용은 아래와 같습니다.

- 소제목에 영문 원문 삽입
- 인수 -> 인자
- 레이어 -> 층
- 히든 -> 은닉

### CODE REVIEW

File: sources/constraints.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-## 제약의 사용법
+## 제약의 사용법<sub>Usage of constraints</sub>
 
-`constraints` 모듈의 함수는 최적화 과정에서 네트워크 매개변수에 제약(예시. 비음수)을 설정할 수 있도록 합니다.
+`constraints` 모듈의 함수는 최적화 과정에서 네트워크 매개변수에 제약(예시: 비음수)을 설정할 수 있도록 합니다.
```
[the remark this answers is not in the document]
Comment (Denny-Hwang) [reply sources/constraints.md]: 네 반영하겠습니다!

File: sources/constraints.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-페널티는 각 레이어 별로 적용됩니다. API의 구체사항은 레이어마다 다를 수 있지만, `Dense`, `Conv1D`, `Conv2D` 그리고 `Conv3D` 레이어는 통일된 API를 가집니다.
+페널티는 각 층별로 적용됩니다. API의 구체사항은 층마다 다를 수 있지만, `Dense`, `Conv1D`, `Conv2D` 그리고 `Conv3D` 층은 통일된 API를 가집니다.
 
-이러한 레이어들은 2가지 키워드 인수를 노출합니다:
+이러한 층들은 2가지 키워드 인자를 노출합니다.
```
[the remark this answers is not in the document]
Comment (Denny-Hwang) [reply sources/constraints.md]: 네 감사합니다. 원문에서 arguments로 쓰여서 '인수'대신 '인자'로 바꿔서

-> '키워드 인자를 가집니다.' 로 수정하겠습니다.

File: sources/constraints.md [hunk 3]
```diff
@@ -71,16 +72,16 @@ keras.constraints.NonNeg()
 keras.constraints.UnitNorm(axis=0)
 ```
 
-각 히든 레이어 유닛에 대응하는 가중치가 단위 노름을 가지도록 제약합니다.
+각 은닉 유닛에 대응하는 가중치가 단위 노름을 가지도록 제약합니다.
```
[the remark this answers is not in the document]
Comment (Denny-Hwang) [reply sources/constraints.md]: 원문에서 'hidden unit'으로 적혀 있었는데요, 이전 번역자분께서 '히든 레이어'로 번역하셨습니다. 그래서 우리 단어논의에 맞게 '은닉 유닛'으로 수정하였습니다. 이건 유지하도록 하겠습니다. 감사합니다.

File: sources/constraints.md [hunk 4]
```diff
@@ -98,10 +99,10 @@ keras.constraints.MinMaxNorm(min_value=0.0, max_value=1.0, rate=1.0, axis=0)
 
 MinMaxNorm 가중치 제약.
 
-각 히든 레이어에 대응하는 가중치를 제약해서
+각 은닉 유닛에 대응하는 가중치를 제약해서
```
[the remark this answers is not in the document]
Comment (Denny-Hwang) [reply sources/constraints.md]: 원문에서 'hidden unit'으로 적혀 있었는데요, 이전 번역자분께서 '히든 레이어'로 번역하셨습니다. 그래서 우리 단어논의에 맞게 '은닉 유닛'으로 수정하였습니다. 이건 유지하도록 하겠습니다. 감사합니다.

**Comment by fuzzythecat** [comment]
감사합니다!
